plugins/secom_cl.py

```diff
@@ -1,24 +1,39 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """
-Created on 26 Jun 2013
-Edited June 2019
+Created on 26 Jun 2013, updated June 2019.
 
-@author: Éric Piel, Sabrina Rossberger
+@author: Éric Piel, Lennard Voortman, Sabrina Rossberger
 
-This is a script to acquire a set of images from the CCD from various e-beam
-spots on the sample along a grid.
-Can also be used as a plugin.
+This is an Odemis plugin to acquire a set of images from the CCD from various
+e-beam spots on the sample along a grid.
 
-
-run as:
-./secom_cl --xrep 45 --yrep 5 --prefix filename-prefix
-
---prefix indicates the beginning of the filename.
 The files are saved in TIFF, with the y, x positions of the ebeam (in nm) in the name,
-the total number of ebeam positions in x and y, the physical distance between positions in x and y
-and the type of the acquisition.
```

Comment:
Why is the part on how to run the script + the options deleted here?

Reply:
That explained (briefly) in the commit text. This functionality was removed 8 months ago. So it makes no sense to still document it.